db/schema.rb

```diff
@@ -0,0 +1,37 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# Note that this schema.rb definition is the authoritative source for your
+# database schema. If you need to create the application database on another
+# system, you should be using db:schema:load, not running all the migrations
+# from scratch. The latter is a flawed and unsustainable approach (the more migrations
+# you'll amass, the slower it'll run and the greater likelihood for issues).
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 20170502015702) do
+
+  create_table "customers", force: :cascade do |t|
```

Comment:
Thanks @case-eee ! I think Josh was trying to implement some of his newfound SQL knowledge from over break and mentioned that it's helpful to make sure you don't have repetition of column names, but does ActiveRecord require updated_at and created_at to be named those things to properly update them? Make sense!